bitmark-references/security.markdown

```diff
@@ -0,0 +1,73 @@
+# Security feature of the Bitmark Blockchain
+
+## Transactions
+
+There are two types of transactions:
+* Single signature - for most basic transactions
+* Countersigned - for special transfers
+
+The data in a transaction is packed in a specific order, depending on
+the transaction type.  An Ed25519 signature is performed over all data
+in the packed transaction and appended to form the submission to the
+block chain.
+
+In the case of a countersigned transaction, the signature is performed
+over the packed transaction *including* the first signature to form
+the complete transaction.  While this does make the signatures order
+sensitive, it does mean that the receiver of the transfer can verify
+the sender before signing.
+
+The Ed25519 signature algorithm details can be viewed at: https://ed25519.cr.yp.to/
+
+The important features for the Bitmark Block chain are:
+* high security signatures with collision resistance
+* small 32 byte keys
+* signature is only 64 bytes
+* fast verification
+
+## Transaction Identifiers
+
```

Comment:
Again, we want to stick with the topic of security.

Something like: "Transactions are securely incorporated into a block through the SHA3-256 algorithm, which is used to derive a 32-byte identifier for each transaction." ...